specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -5330,6 +5539,76 @@
           "description": "Error when What-If operation fails."
         }
       }
+    },
+    "Tags": {
+      "description": "key and value pairs for tags",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "description": "The tag value."
+          }
+        }
+      }
+    },
+    "TagPatchRequest": {
+      "type": "object",
+      "description": "Tag Request for Patch operation.",
+      "properties": {
+        "operation": {
```

Comment:
should operation & properties be required?

Reply:
> should operation & properties be required?

Added "Required" for "operation" and "properties", then got a LintDiff error: R2016 -PatchBodyParametersSchema: Properties of a PATCH request body must not be required.

So, Removing "Required" for PatchTagRequest.